tests/models/mamba/test_mamba.py

```
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+from transformers import AutoTokenizer
+
+from tests.models.mamba.mamba_ssm.model import Mamba
+from tests.utils import ModelTester, repeat_inputs
+from loguru import logger
+
+torch._dynamo.config.suppress_errors = True
```

Can you remove this line? We generally do not want to suppress any errors. Any error should be thoroughly investigated.

---

I have removed the `torch._dynamo.config.suppress_errors = True` and investigated the error's and shared the insight in this [bounty issue comment](https://github.com/tenstorrent/pytorch2.0_ttnn/issues/1048#issuecomment-3271875658).